manifests/nexus.yaml

```diff
@@ -10,5 +33,5 @@ spec:
   charts:
   - name: cray-nexus
     source: csm-algol60
-    version: 0.11.1
+    version: 0.12.1
```

Comment:
To store images together with Sigstore attachments in Cloud Nexus, we have to upgrade Nexus to 3.67.1 (and consequently upgrade `cray-nexus-setup` image to `0.11.0`).